virtme/guest/virtme-sshd-script

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# Initialize ssh server for remote connections (option `--server ssh`)
+
+HOME=$(getent passwd "${virtme_user}" | cut -d: -f6)
```

Comment:
If I'm not mistaken, `virtme_user` is not set when `root` is used.

Maybe safer to use `${virtme_user:-root}`?

Reply:
The problem with root is that we won't have access to the real root's ssh keys, so that won't work.

Maybe we can always use the user that runs vng on the host (independently on `${virtme_user}`) to make sure that we will always be able to configure sshd as long as the user has some ssh keys available. But for this we need to pass a separate variable, like `${host_user}`.

Reply:
> The problem with root is that we won't have access to the real root's ssh keys, so that won't work.

It looks OK on my side:

```
$ sudo ls /root/.ssh/
authorized_keys
$ sudo ssh-keygen
(...)
$ sudo ls /root/.ssh/
authorized_keys  id_ed25519  id_ed25519.pub
$ sudo vng -r --user root -- ls /root/.ssh/
authorized_keys
id_ed25519
id_ed25519.pub
## and ${virtme_user} is not set
```

> Maybe we can always use the user that runs vng on the host (independently on `${virtme_user}`) to make sure that we will always be able to configure sshd as long as the user has some ssh keys available. But for this we need to pass a separate variable, like `${host_user}`.

Would it not be enough to description the limitations in the README or the `--help` menu? e.g. `sshd` is required, the host keys will be generated, only the current user (or `--user` if set) can access it, the private keys of this user will be added to the `authorized_keys` files (then not recommended with `--rw`), etc.?